todo_app/app.py

```diff
@@ -7,13 +7,28 @@
 from flask_login import LoginManager, login_required, login_user, current_user
 from oauthlib.oauth2 import WebApplicationClient
 from todo_app.user import User
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+
 import requests
 import os
-
+import logging
 
 def create_app():
     app = Flask(__name__)
 
+    #logging.basicConfig(filename='ToDoApp.log', level=logging.INFO, format='%(asctime)s - %(module)s - %(message)s')
+    logging.basicConfig(filename='ToDoApp.log', format='%(asctime)s - %(module)s - %(message)s')
```

Comment:
Nice addition! Worth noting that this stops messages hitting the console, so it might be even nicer to make this additional rather than overriding the basic config.
Also, it would be good to add `ToDoApp.log` to your `.gitignore` now that you have this